zarr/tests/test_core.py

```diff
@@ -62,16 +64,59 @@
 # noinspection PyMethodMayBeStatic
 
 
-class TestArray(unittest.TestCase):
-
+class TestArray():
     version = 2
     root = ''
-    KVStoreClass = KVStore
+    path = ''
+    compressor = Zlib(level=1)
```

Comment:
Curious about why we are setting a default compressor here, and / or why it is different from the built-in default.

Reply:
I did that to make the test pass :) My observation is that historically, different test classes used different compressors (I have no idea why), and because of the hexdigest tests, these compressors get immured with the rest of the tests in that test class. I think this alone argues for rethinking the hexdigest tests.